shared/utility/input/ServoID.hpp

```diff
@@ -47,7 +47,9 @@ namespace utility::input {
             L_ANKLE_ROLL     = 17,
             HEAD_YAW         = 18,
             HEAD_PITCH       = 19,
-            NUMBER_OF_SERVOS = 20
+            L_FOOT_BASE           = 20,
+            R_FOOT_BASE           = 21,
+            NUMBER_OF_SERVOS = 22
```

Comment:
but we only have 20 servos!

Reply:
Hmm, we could add something like LinkID if we want to keep ServoID's to only include the actuatable servo joints. With the URDF we can add fixed joints/frames to our model which is useful for specifying additional frames of interest such as the base of the foot, hands, camera locations etc.